README.md

```diff
@@ -78,6 +82,8 @@ It takes 5 entries for 3 happy days to occur
 
 **think**: What are you going to do if there aren't at least 3 happy days? Where do you need to handle that case?
 
+I decided to have an alternative string to puts when the total entries didn't include at least 3 happy days. I put it at the end of my overall_mood method.
+
```

Comment:
This works. Because ruby. We will hopefully discuss this in class. It's weird to me as a Java programmer because it means the method could return different data types.